packages/dashboard/src/portal/lib/farms.ts

```diff
@@ -84,6 +85,12 @@ export async function getFarmPayoutV2Address(
   const address = await api.query.tfgridModule.farmPayoutV2AddressByFarmID(id);
   return address.toJSON();
 }
+
+export async function getNodesCount(api: { query: any }, farmID: number) {
+  const nodes = await axios.get(`${config.gridproxyUrl}/nodes?farm_ids=${farmID}`);
```

Comment:
I think we should get the nodes' count from the ret_count header instead as the farm may has more than 50 nodes